src/wallet/rpc/spend.cpp

```diff
@@ -1472,6 +1490,18 @@ RPCHelpMan sendall()
             const CAmount fee_from_size{fee_rate.GetFee(tx_size.vsize)};
             const CAmount effective_value{total_input_value - fee_from_size};
 
+            // Cap tx weight based on estimated size if requested
+            if (options.exists("max_tx_weight")) {
+                const auto max_tx_weight =  options["max_tx_weight"].getInt<int>();
+                if (max_tx_weight < 0 || max_tx_weight > MAX_STANDARD_TX_WEIGHT) {
```

Comment:
Does 0 even ever make sense?

Reply:
as much a sense as "1". Could have minimal relay size here instead?

Reply:
At second glance, lemme retract my comment. It doesn’t really hurt to allow values down to 0, even if that is less than a sensible use of that parameter. Might be helpful in tests to be able to restrict it fairly low.